docs/sql-reference/sql-statements/data-definition/CREATE REPOSITORY.md

```diff
@@ -62,6 +62,13 @@ StarRocks supports creating repositories in HDFS, AWS S3, and Google GCS.
 >
 > StarRocks supports creating repositories in Google GCS only according to the S3A protocol. Therefore, when you create repositories in Google GCS, you must replace the prefix in the GCS URI you pass as a repository location in `ON LOCATION` with `s3a://`.
 
+
+- For Aliyun OSS:
+  - "fs.oss.accessKeyId": The Access Key ID that you can use to access the Aliyun OSS bucket. 
+  - "fs.oss.accessKeySecret": The Secret Access Key that you can use to access the Aliyun OSS bucket.
+  - "fs.oss.endpoint": The endpoint that you can use to access the Aliyun OSS bucket.
```

Comment:
```suggestion
  - "fs.oss.endpoint": The endpoint used to access the Aliyun OSS bucket.
```

A little shorter

Reply:
Thank you for your suggestion. I have updated the wording here based on your feedback, and also made similar adjustments in other places with the same issue.